manifest.toml

```
@@ -0,0 +1,74 @@
+packaging_format = 2
+
+id = "nextcloud"
+name = "Nextcloud"
+description.en = "Online storage, file sharing platform and various other applications"
+description.fr = "Stockage en ligne, plateforme de partage de fichiers et diverses autres applications"
+
+version = "25.0.1~ynh1"
+
+maintainers = ["kay0u"]
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://nextcloud.com"
+demo = "https://demo.nextcloud.com/"
+admindoc = "https://docs.nextcloud.com/server/stable/admin_manual/"
+userdoc = "https://docs.nextcloud.com/server/latest/user_manual/en/"
+code = "https://github.com/nextcloud/server"
+cpe = "cpe:2.3:a:nextcloud:nextcloud"
+
+[integration]
+yunohost = ">= 11.1.6"
+architectures = ["amd64", "arm64"]
+multi_instance = true
+ldap = true
+sso = true
+disk = "50M"
+ram.build = "128M"
+ram.runtime = "512M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/nextcloud"
+
+    [install.admin]
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "Set to 'visitors', Nextcloud will be accessible by Nextcloud Desktop and by users without a YunoHost account. This can be changed later in the webadmin."
+    help.fr = "Défini sur 'visiteurs', Nextcloud sera accessible par Nextcloud Desktop et par les utilisateurs n’ayant pas de compte YunoHost. Vous pourrez changer dans la webadmin."
+    type = "group"
+    default = "visitors"
+
+    [install.user_home]
+    ask.en = "Access the users home folder from Nextcloud?"
+    ask.fr = "Accéder au dossier personnel des utilisateurs depuis Nextcloud ?"
+    type = "boolean"
+    default = false
+
+[resources]
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.data_dir]
+
+    [resources.permissions]
+    main.url = "/"
+    
+    api.url = "re:$domain\\/.well-known\\/.*"
```

```suggestion
    api.url = "re:__DOMAIN__\\/.well-known\\/.*"
```

i'm pushing a change in the core to support this syntax

(Also i'm not sure if we really need a regex here or if `__DOMAIN__/.well-known/` could be enough